int16 id
string name
geometry_msgs/Pose position